client/src/main/java/com/battleships/connection/commands/server/commands/HitMessage.java

```diff
@@ -0,0 +1,24 @@
+package com.battleships.connection.commands.server.commands;
+
+import com.battleships.RawBoardField;
+import com.battleships.connection.Connection;
+import com.battleships.connection.commands.AbstractServerCommand;
+import javafx.application.Platform;
+
+import java.util.List;
+
+public class HitMessage extends AbstractServerCommand {
+    public HitMessage(Object value) {
+        super(value);
+    }
+
+    @Override
+    public void execute() {
```

Comment:
extract execute content to separate methods with descriptive names what happens inside.
Execute() is ok, because it is overrided method but I do not know what happens inside it

Reply:
fixed